src/math.rs

```diff
@@ -36,10 +36,19 @@ pub fn lerp(source: f32, target: f32, weight: f32) -> f32 {
 pub fn remap<T, U>(value: T, source_range: &U, target_range: &U) -> Option<f64>
 where
     f64: std::convert::From<T>,
```

Comment:
Not saying you should change this, but note that this can also be written as `T: std::convert::Into<f64>`. In general, if there is a `From` implementation, the `Into` implementation gets generated automatically, because the standard library provides this blanket implementation: https://doc.rust-lang.org/stable/src/core/convert/mod.rs.html#553-560

Reply:
I tried, didn't work for some reason. Maybe I did something differently, but that was my first attempt. Also I kind of like it this way because it describes better what it actually does.